scripts/validate-build.sh

```
@@ -0,0 +1,490 @@
+        fi
+
+        log_success "All webpack chunks present"
+    else
+        log_info "No remoteEntry.js found (no federated modules configured)"
+    fi
+
+    # Check for large chunks
+    LARGE_CHUNKS=$(find /tmp/dist-check -name "*.bundle.js" -size +1M 2>/dev/null || true)
+    if [ -n "$LARGE_CHUNKS" ]; then
+        log_warning "Large chunks detected (may slow page load):"
+        du -h $LARGE_CHUNKS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)